src/input/label.component.ts

```diff
@@ -38,6 +38,7 @@ import { TextArea } from "./text-area.directive";
 	template: `
 		<label
 			[for]="labelInputID"
+			[attr.aria-label]="labelInputID"
```

Comment:
Generally we want to add a `ariaLabel` `@Input` for this 👍 The `labelInputID` is just to identify this label, it's not meant to be human readable.

Reply:
Thanks, Updated it has Input.